specification/src/specification/queries/field-selection.md

```diff
@@ -7,6 +7,28 @@ A [`Query`](../../reference/types.md#query) can specify which fields to fetch. T
 
 The requested fields are specified as a collection of [`Field`](../../reference/types.md#field) structures in the `field` property on the [`Query`](../../reference/types.md#query).
 
+## Field Arguments
+
+Arguments can be supplied to fields via the `arguments` key. These match the format described in [the arguments documentation](../arguments.md).
+
+The [schema response](../schema/object-types.md) will specify which fields take arguments via its respective `arguments` key.
+
+- If a field has any arguments specified, then all arguments must be provided when that field is queried
+- A literal argument `null` value may be used for optional arguments
```

Comment:
Let's not use the word optional because it'll be confusing (required/optional vs nullable/non-nullable). We could say all arguments are required, including nullable arguments.

Reply:
```
- If a field has any arguments specified, then all arguments must be provided when that field is queried
- Literal argument `null` values may be used for nullable arguments
```

Reply:
Is this ok?